docs/troubleshooting-oauth2.md

```diff
@@ -68,15 +75,16 @@ OAuth resource [rabbitmq] not available. OpenId Discovery endpoint https://<the_
 
 #### Troubleshoot the issue
 
-This issue is caused when the endpoint is not returning a JSON payload which matches with the [OpenId Connect Discovery Configuration](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig). 
-There are the possible causes:
-- The payload returned by the endpoint is not compliant because it is empty or it is missing some critical information. To identify the root cause, you open the browser's javascript console and search for one of these possible error messages:
+This error means that the endpoint does not return a JSON payload which matches with the [OpenId Connect Discovery Configuration](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig) format.
+
+There are several the possible causes:
+
+ * The payload returned by the endpoint is not compliant because it is empty or it is missing some critical information. To identify the root cause, you open the browser's javascript console and search for one of these possible error messages:
   - `Payload does not contain openid configuration` This error occurs when the payload is empty or it is not a JSON payload
   - `Missing authorization_endpoint` This error occurs when the JSON attribute `authorization_endpoint` is missing
   - `Missing token_endpoint` This error occurs when the JSON attribute `token_endpoint` is missing
   - `Missing jwks_uri` This error occurs when the JSON attribute `jwks_uri` is missing
-  
```

Comment:
Maybe add full stops at the end of the bullet points above, again, I would search for what is in general in OSS docs for bullet lists, is there full stops at the end of them or not, go with what is most consistent